diff --git a/node_modules/@remix-run/serve/index.js b/node_modules/@remix-run/serve/index.js
index ff63316..d69b901 100644
--- a/node_modules/@remix-run/serve/index.js
+++ b/node_modules/@remix-run/serve/index.js
@@ -26,7 +26,28 @@ var morgan__default = /*#__PURE__*/_interopDefaultLegacy(morgan);
 function createApp(buildPath, mode = "production") {
   let app = express__default["default"]();
   app.disable("x-powered-by");
-  app.use(compression__default["default"]());
+  app.use(compression__default["default"]({
+    filter: (req, res) => {
+      let contentTypeHeader = res.getHeader("Content-Type");
+      let contentType = "";
+      if (contentTypeHeader) {
+        if (Array.isArray(contentTypeHeader)) {
+          contentType = contentTypeHeader.join(" ");
+        } else {
+          contentType = String(contentTypeHeader);
+        }
+      }
+
+      if (
+        contentType.includes("text/html") ||
+        contentType.includes("text/event-stream")
+      ) {
+        return false;
+      }
+
+      return true;
+    },
+  }));
   app.use("/build", express__default["default"].static("public/build", {
     immutable: true,
     maxAge: "1y"
